src/Place/DuplicatePlace/Dto/ClusterRecord.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\DuplicatePlace\Dto;
+
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+class ClusterRecord
+{
+    private string $clusterId;
+
+    private UuidInterface $placeUuid;
+
+    public function __construct(string $clusterId, UuidInterface $placeUuid)
```

Comment:
If this is a wrapper around a cluster I would expect it to have a list of places instead of just one place. This feels more like a `PlaceInCluster` row

Reply:
This class has been deleted.